approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "scienzecognitive"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "cultura"
+- "d"
+- "database"
+- "debugging"
+- "design"
+- "devops"
+- "sistemidistribuiti"
+- "dotnet"
+- "dragonflybsd"
+- "istruzione"
+- "elixir"
+- "elm"
+- "emacs"
+- "erlang"
+- "evento"
+- "metodiformali"
+- "fortran"
+- "freebsd"
+- "giochi"
+- "go"
+- "grafica"
+- "hardware"
+- "haskell"
+- "storia"
+- "illumos"
+- "ios"
+- "java"
+- "javascript"
+- "lavoro"
+- "legge"
+- "linux"
+- "lisp"
+- "lua"
+- "mac"
+- "matematica"
+- "meta"
+- "ml"
+- "mobile"
+- "netbsd"
+- "reti"
+- "networking"
+- "nodejs"
+- "objectivec"
+- "openbsd"
+- "sistemioperativi"
+- "osdev"
+- "pdf"
+- "performance"
+- "perl"
+- "persona"
+- "filosofia"
+- "php"
+- "plt"
+- "pratiche"
+- "privacy"
+- "programmazione"
+- "python"
+- "lamentela"
+- "rilascio"
+- "inversione"
+- "ruby"
+- "rust"
+- "satira"
+- "scala"
+- "scalabilità"
+- "scienza"
+- "sicurezza"
+- "mostra"
+- "presentazione"
```

Comment:
come sopra

Reply:
Come per PDF, non potrebbe indicare che il link punta a delle slides e non ad una pagina di testo?

Reply:
Nel caso in caso metterei PPT

Reply:
`"slides"` mi sembra più appropriato allora.